modules/end-user-guide/pages/adding-a-vscode-extension.adoc

```diff
@@ -1,24 +1,34 @@
-:navtitle: Adding a Visual Studio Code Extension
+:navtitle: Adding a Visual Studio Code extension to a workspace
 :keywords: Visual Studio Code extension, user-guide
 
 [id="adding-visual-studio-code-extension_{context}"]
-= Adding a Visual Studio Code Extension
+= Adding a Visual Studio Code extension to a workspace
 
-IDE specific plug-ins are no longer part of devfile with devfile v2. Instead, it uses the specific meta-folder to describe the plug-ins.
+Devfiles in the devfile v2 format don't contain any IDE-specific plug-ins. Instead, the plug-ins are described in a specific meta-folder.
```

Comment:
Devfiles in the devfile v2 format don't contain any IDE-specific plug-ins or Visual Studio Code extensions. Instead, the plug-ins and extensions are described in a specific meta-folder.